static/index.html

```diff
@@ -35,14 +40,17 @@
       </ul>
 
       <div class="connection-actions">
-        <a href="#" id="edit_connection" class="btn btn-default btn-sm"><i class="fa fa-database"></i> Connect</a>
-        <a href="#" id="close_connection" class="btn btn-default btn-sm">Disconnect</a>
+        <a href="#" id="edit_connection" class="btn btn-default text-white btn-sm"><i
```

Comment:
Are you using some kind of auto formatter for html here? The tag content looks broken over the lines